pom.xml

```diff
@@ -1,12 +1,97 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <groupId>com.epam.training.center.qa</groupId>
     <artifactId>AutoTestProj</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
```

Comment:
remove pls .gitkeep that you don't need anymore, for example in test/java and test/resources